### PR TITLE
Merge SLE-15-SP2 into master

### PR DESCRIPTION
**Pull request by teclator**
It merges #1121

### CODE REVIEW

File: package/yast2-network.changes [hunk 1]
```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Mon Nov 23 08:03:17 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- Do not show a warn message when modifying a bonding configuration
+  and all the slaves are already configured with BOOTPROTO='none'
+  (bsc#1178950)
+- 4.2.85
```
Comment (imobachgs) [inline package/yast2-network.changes:7]: Version number looks wrong (it should be ' 4.3.29`).

**Review by imobachgs — CHANGES_REQUESTED** [review]

File: package/yast2-network.changes (hunk 1 above)
Comment (teclator) [inline package/yast2-network.changes:7]:
```suggestion
- 4.3.29
```

**Review by imobachgs — APPROVED** [review]

File: package/yast2-network.changes (hunk 1 above)
Comment (teclator) [reply package/yast2-network.changes]: Yep, I adapted the spec but do not touched the changes, as apparently I removed a line when I suggested the changes I will merge it again locally and will force the push.

**Comment by teclator** [comment]
@imobachgs it should be ok now.

**Review by imobachgs — APPROVED** [review]

**Pull request merged**